### DISCUSSION TITLE
How to parse a AIVDM/AIVDO?

### DISCUSSION THREAD

**Discussion by smallq3868** [Knio/pynmea2#137] [Q&A]
First of all, Thank you for sharing your code.
By the way, I need to parse a type "AIVDM/AIVDO". But in this code, that type be regonized by Errorparse code.

How to add talker this type.

Thank you.

**Comment by Knio** [comment]
See: #84 
I worked a bit on a prototype of this, but AIS messages turned out to be more complicated than all of the rest of NMEA combined, so I think you should just use a dedicated AIS library for this rather than try to add it to pynmea2
